src/routes/wallet.routes.js

```diff
@@ -0,0 +1,380 @@
+            return false;
+          }
+          return true;
+        }
+      }
+    },
+    change_address: {
+      in: ['body'],
+      isString: true,
+      optional: true
+    },
+    debug: {
+      in: ['body'],
+      isBoolean: true,
+      toBoolean: true,
+      optional: true,
+    }
+  }),
+  sendTx
+);
+
+/**
+ * POST request to create a token
+ * For the docs, see api-docs.js
+ */
+walletRouter.post(
+  '/create-token',
+  body('name').isString(),
+  body('symbol').isString(),
+  body('amount').isInt({ min: 1 }).toInt(),
+  body('address').isString().optional(),
+  body('change_address').isString().optional(),
+  createToken
+);
+
+/**
+ * POST request to mint tokens
+ * For the docs, see api-docs.js
+ */
+walletRouter.post(
+  '/mint-tokens',
+  body('token').isString(),
+  body('amount').isInt({ min: 1 }).toInt(),
+  body('address').isString().optional(),
+  body('change_address').isString().optional(),
+  mintTokens
+);
+
+/**
+ * POST request to melt tokens
+ * For the docs, see api-docs.js
+ */
+walletRouter.post(
+  '/melt-tokens',
+  body('token').isString(),
+  body('amount').isInt({ min: 1 }).toInt(),
+  body('change_address').isString().optional(),
+  body('deposit_address').isString().optional(),
+  meltTokens
+);
+
+/**
+ * GET request to filter utxos before consolidation
+ * For the docs, see api-docs.js
+ */
+walletRouter.get(
+  '/utxo-filter',
+  query('max_utxos').isInt().optional().toInt(),
+  query('token').isString().optional(),
+  query('filter_address').isString().optional(),
+  query('amount_smaller_than').isInt().optional().toInt(),
+  query('amount_bigger_than').isInt().optional().toInt(),
+  query('maximum_amount').isInt().optional().toInt(),
+  query('only_available').isBoolean().optional().toBoolean(),
+  utxoFilter
+);
+
+/**
+ * POST request to consolidate utxos
+ * For the docs, see api-docs.js
+ */
+walletRouter.post(
+  '/utxo-consolidation',
+  body('destination_address').isString(),
+  body('max_utxos').isInt().optional().toInt(),
+  body('token').isString().optional(),
+  body('filter_address').isString().optional(),
+  body('amount_smaller_than').isInt().optional().toInt(),
+  body('amount_bigger_than').isInt().optional().toInt(),
+  body('maximum_amount').isInt().optional().toInt(),
+  utxoConsolidation
+);
+
+/**
+ * POST request to create an NFT
+ * For the docs, see api-docs.js
+ */
+walletRouter.post(
+  '/create-nft',
+  body('name').isString(),
+  body('symbol').isString(),
+  body('amount').isInt({ min: 1 }).toInt(),
+  body('data').isString(),
+  body('address').isString().optional(),
+  body('change_address').isString().optional(),
+  body('create_mint').isBoolean().optional().toBoolean(),
+  body('create_melt').isBoolean().optional().toBoolean(),
+  createNft
+);
+
+/**
+ * POST request to stop a wallet
+ * For the docs, see api-docs.js
+ */
+walletRouter.post('/stop', stop);
+
+walletRouter.use('/tx-proposal', txProposalRouter);
```

Comment:
Maybe put this after the middleware on the start of the file.

Reply:
Done on [e0b7301](https://github.com/HathorNetwork/hathor-wallet-headless/pull/174/commits/e0b7301ba3e5148be371bb786c9c622919049655)